homeassistant/components/minecraft_server/quality_scale.yaml

```diff
@@ -0,0 +1,113 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration doesn't provide any service actions.
+  appropriate-polling:
+    status: done
+    comment: Default polling interval is set to 60s.
+  brands: done
+  common-modules: done
+  config-flow: done
+  config-flow-test-coverage: done
```

Comment:
Let's have every test end in either CREATE_ENTRY or ABORT to test we are able to recover from errors and are able to finish the flow

Reply:
Oh I see that you have `test_recovery`, but let's do it with all test and parametrize where possible